Packs/CommonPlaybooks/Playbooks/playbook-CVE_Enrichment_-_Generic_v2.yml

```
@@ -563,9 +614,9 @@ outputs:
   description: Vendor ID.
 - contextPath: VulnDB.Products.Id
   description: Product IDs.
-sourceplaybookid: cve_enrichment_-_generic
 tests:
-- No test
-marketplaces:
-  - xsoar
-  - marketplacev2
+- No tests (auto formatted)
+contentitemexportablefields:
+  contentitemfields:
+    propagationLabels: []
+system: true
```

Please re-add that Marketplaces
```suggestion
system: true
marketplaces:
  - xsoar
  - marketplacev2
```